pkg/kv/kvserver/asim/tests/testdata/non_rand/example_fulldisk

```diff
@@ -19,46 +19,46 @@ OK
 # store 5. This is shown below as it sheds replicas.
 plot stat=replicas
 ----
- 342 ┤                                                                    ╭╮╭─╮╭╭╭─╮╭
- 331 ┤                         ╭╮ ╭╭─╮╮╭╮╭╭╮╭╭───╮─╭──╮╮  ╭╮──╭╭───╮─╮╭──╮╯╭╯╭╰╮╭╯│╰─
- 319 ┤                  ╭╭╮╭╮─╭─╭──╯╭│╭╭─────╯╯╰─│╭╯╮╭│╰─╭╮╭───╯╭─╯│╭─╯╭─╰─╯╰╯─╰╯╭╰╰╯
- 308 ┤               ╭───╭──────╯╯╰╰╯╰─╯╰╯  ╰╯   ╰╯ ╰╯╰──╯╰╯  ╰╰╯╯ ╰╯ ╰╯        ╰╯
- 296 ┼───────────────╮───╯╯╯╰╯
- 285 ┤               ╰──╮
- 274 ┤                  │╭─╮
- 262 ┤                  ╰╯ │╭╮
- 251 ┤                     ╰╯╰╮ ╭╮
- 239 ┤                        │╭╯│                     ╭─╮
- 228 ┤                        ╰╯ ╰╮╭╮   ╭╮ ╭─╮         │ │ ╭╮  ╭╮
- 217 ┤                            ╰╯╰───╯│╭╯ │   ╭╮ ╭╮╭╯ ╰─╯╰╮╭╯│     ╭╮
- 205 ┤                                   ╰╯  ╰╮╭─╯╰─╯╰╯      ╰╯ ╰╮ ╭╮ ││
- 194 ┤                                        ││                 ╰─╯╰─╯╰╮╭╮ ╭╮  ╭╮ ╭╮
- 182 ┤                                        ╰╯                        ╰╯╰╮│╰╮╭╯│ │╰
- 171 ┤                                                                     ╰╯ ╰╯ ╰─╯
+ 364 ┤       ╭╭╮╭─╮╮╭╭╮─╭───╮╭─╮╮╭─╭╮───╭╮─╭╭─╮─╭────╮╭╭───────╮──╭─╮─╭──────────────
+ 344 ┤    ╭───╯╭─────╯╰─╯───╰──────╯╰───╯╰──╯╯╰─╯╰╰──╰─╯╯╰────╯╰──╯╯╰─╯╯  ╰───╯ ╰────
+ 324 ┤  ╭─╯╭───╯
+ 304 ┼─╮───╯
+ 283 ┤ ╰╮
+ 263 ┤  │
+ 243 ┤  ╰╮
+ 223 ┤   ╰╮
+ 203 ┤    ╰╮
+ 183 ┤     ╰╮
+ 163 ┤      ╰╮
+ 143 ┤       │
+ 122 ┤       ╰╮
+ 102 ┤        ╰────────╮╭──╮╭─╮╭──╮ ╭──╮ ╭──╮ ╭╮
+  82 ┤                 ╰╯  ╰╯ ╰╯  ╰─╯  ╰─╯  ╰─╯╰──────────────╮╭───╮╭───╮╭────╮ ╭──╮
+  62 ┤                                                        ╰╯   ╰╯   ╰╯    ╰─╯  ╰─
                                           replicas
 initial store values: [s1=300, s2=300, s3=300, s4=300, s5=300] (stddev=0.00, mean=300.00, sum=1500)
-last store values: [s1=342, s2=341, s3=326, s4=328, s5=178] (stddev=62.84, mean=303.00, sum=1515)
+last store values: [s1=363, s2=352, s3=360, s4=362, s5=66] (stddev=117.36, mean=300.60, sum=1503)
 
 # Plot the % of disk storage capacity used. We should see s5 hovering right
 # around 92.5-95% (the storage capacity threshold value).
 plot stat=disk_fraction_used
 ----
- 1.10 ┤                                     ╭╮          ╭╮             ╭╮           ╭╮
- 1.04 ┤                      ╭╮  ╭╮         │╰╮   ╭╮    ││  ╭╮  ╭╮     ││ ╭╮ ╭╮ ╭─╮ │╰
- 0.97 ┤           ╭───╮╭─╮ ╭╮│╰╮╭╯╰╮╭╮╭╮ ╭╮ │ ╰╮╭╮││ ╭╮ │╰╮╭╯╰╮╭╯╰╮ ╭╮ │╰╮││ │╰╮│ │╭╯
- 0.91 ┤    ╭──────╯   ╰╯ ╰─╯╰╯ ╰╯  ╰╯╰╯╰─╯│╭╯  ││╰╯╰─╯│╭╯ ╰╯  ╰╯  ╰─╯╰─╯ ╰╯╰─╯ ╰╯ ╰╯
- 0.84 ┼────╯                              ╰╯   ╰╯     ╰╯
- 0.78 ┤
- 0.71 ┤
- 0.65 ┤
- 0.59 ┤
- 0.52 ┤
- 0.46 ┤
- 0.39 ┤
- 0.33 ┤
- 0.26 ┤                                               ╭╮  ╭╮╮╭─╭╭────╮╭───────────────
- 0.20 ┤               ╭╮╭╭────────────────────────────╯╰──╯╰────╯─╯╰─╰╯──╯
- 0.14 ┼──────────────────╯ ╰╯╰╯
+ 2.51 ┼─╮
```

Comment:
Something changed significantly here - instead of hovering around 92%, it starts out at 251%. Something to look into here.